docs/getting-started/products/yvaults/vault-tokens.md

```diff
@@ -25,7 +25,7 @@ The vault tokens have evolved between v1, v2, and v3, here are a couple of main
 
 ## V3 yVault Tokens
 
-- The strategy contract is [ERC-4626](https://ethereum.org/en/developers/docs/standards/tokens/erc-4626/) compliant (we call it a "Tokenized Strategy").
-- A strategy can act as a vault with only 1 strategy.
-- A strategy can act as a vault for many strategies.
-- Vault tokens are still prefixed with `yv`, so a v3 vault for USDC gives the user yvUSDC.
\ No newline at end of file
+- All V3 vaults are [ERC-4626](https://ethereum.org/en/developers/docs/standards/tokens/erc-4626/) compliant.
+- Vaults can be either multi-strategy or single strategy (Tokenized Strategy).
```

Comment:
it is unclear what "Tokenized Strategy" refers to here. Is it only for the single strategy? And what is that?

Reply:
is the differentiator here between "Allocator Vaults" and "Tokenized Strategies"?

Reply:
Maybe the best thing to do here is either add both or neither. So either:

> "Vaults can be either multi-strategy [Allocator Vaults](link to definition) or single-strategy [Tokenized Strategies](link to definition)."

or

> "Vaults can be either multi-strategy or single-strategy."